pkg/extension/opampextension/README.md

```diff
@@ -31,6 +31,9 @@ It manages:
   configuration received from the OpAMP server. This directory must coincide
   with a configuration provider (e.g. glob) for the configuration to be loaded
   by the collector.
+- `disable_tag_new_config_merge`: (optional) Boolean flag to disable new config merge
+  flow introduced for tag edit feature for remotely managed collectors. Default
+  value is disabled.
```

Comment:
Nit: default value - false.

Also, how does this name sound - `disable_tag_replacement`. Since we're replacing tags (under collector_fields) correct?

Reply:
In general, we are introducing a new config merge flow and tag replacement is a more concrete term. In future, we may have this different merge flow for other parameters as well if required. so i think the general term would be the new config merge flow.

Reply:
"New config merge" doesn't mean much to the customer and I don't think we'll be doing this for other collector fields, since that would most likely break collector configuration. From the customer's point of view it helps to be specific about how the field changes their config. If it replaces their tags, that's what the config field could say and if disables tag replacement the field could be called the same.

Reply:
Changed the param name @rnishtala-sumo

Reply:
One more comment, thanks for your patience.